CONTRIBUTING.md

```diff
@@ -43,36 +46,31 @@ print(tree.pretty())
 ```
 
 ```shell
-start
+filter
   expression
-    term
-      term
-        atom
-          comparison
-            _mp_bandgap
-            >
-            5.0
-      AND
-      atom
+    expression_clause
+      expression_phrase
         comparison
-          _cod_molecular_weight
-          <
-          350
-```
-
-```python
-# Assumes graphviz installed on system (e.g. `conda install -c anaconda graphviz`) and `pip install pydot`
-from lark.tree import pydot__tree_to_png
-
-pydot__tree_to_png(tree, "exampletree.png")
```

Comment:
Is this not possible anymore? Or do you think it doesn't matter?

Reply:
As the grammar has updated, the tree it makes now is horrid and way more nested, so thought it best to remove it... the instructions also didn't quite work for me